bootstrap/sql/mysql/v003__create_db_connection_info.sql

```diff
@@ -290,3 +290,5 @@ DROP COLUMN deactivated,
 ADD COLUMN deleted BOOLEAN GENERATED ALWAYS AS (JSON_EXTRACT(json, '$.deleted')),
 ADD INDEX (deleted);
 
+UPDATE policy_entity
```

Comment:
@sureshms  I'm pretty new to flyway
Making these changes based on my limited understanding.